pkg/gemmlowp/Makefile

```diff
@@ -6,4 +6,4 @@ PKG_LICENSE=Apache2.0
 include $(RIOTBASE)/pkg/pkg.mk
 
 all:
-	"$(MAKE)" -C $(PKG_BUILDDIR) -f $(CURDIR)/Makefile.$(PKG_NAME)
+	@:
```

Comment:
never seen this, does it have an advantage over `@true`?

Reply:
I read [on stackoverflow](https://stackoverflow.com/a/26923107) that the noop was faster.